include/vikunja/reduce/detail/BlockThreadReduceKernel.hpp

```diff
@@ -104,11 +104,10 @@ namespace vikunja
 
                     using MemPolicy = TMemAccessPolicy;
                     // Create an iterator with the specified memory access policy that wraps the input iterator.
-                    vikunja::mem::iterator::PolicyBasedBlockIterator<MemPolicy, TAcc, TInputIterator> iter(
-                        source,
-                        acc,
-                        n,
-                        TBlockSize);
+                    using MenIndex = vikunja::MemAccess::BlockStrategy<MemPolicy, TAcc, TIdx>;
```

Comment:
Is `MenIndex` (instead of `MemIndex` intentional)?

Reply:
You are right. I fixed it.